### ISSUE TITLE
Improve masking of fluorescence data in preprocessing

### ISSUE THREAD

**Issue by Soorya19Pradeep** [mehta-lab/microDL#199]
We need better fluorescence image segmentation which works well for the subcellular organelles, especially nucleus and membrane stains. Current masks are not indicative of the segmented fluorescence images. Image of segmented mask overlayed on the corresponding nuclear image attached below.

![image](https://user-images.githubusercontent.com/101817974/218488470-14b74a1f-101f-4a41-8e2d-7ae0c8510590.png)

**Comment by Soorya19Pradeep** [comment]
I have replaced the existing otsu thresholding code with opencv otsu thresholding and here is a FOV with the stain and the result overlaying the mask on the nuclear fluorescence image.
![image](https://user-images.githubusercontent.com/101817974/218541382-0fbc1288-2ccf-46e6-be63-6c715c274e7a.png)
![image](https://user-images.githubusercontent.com/101817974/218541545-5bedbdcc-0be3-4346-984f-ada6bb989a2e.png)

**Comment by Soorya19Pradeep** [comment]
I have added mask generation operation for structures like cell membrane stain. It uses Laplacian filtering of the fluorescence stain as @mattersoflight instructed and then thresholding and cleanup for segmentation. I have tested the preprocessing on two cases, nucleus and membrane masking. The membrane mask generated currently looks like this:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/101817974/219465964-f7c7f73a-550a-49ee-95c0-de28d6d1165e.png">
